IAMOTZ/ievents
# #152797395 Implement the edit center page with react

#### What does this PR do?
It uses react and redux to implement the edit-center page
#### Description of Task to be completed?
User should be able to modify the details of a center
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- Run `npm run react:dev`
- visit the port where the webpack-dev-server started
- log in as an admin
- add some sample centers
- navigate to the centers page
- click to edit any center
- you should be redirected to another page with a form containing the previous details of the center
#### Any background context you want to provide?
- The frontend still uses the locally hosted server, so you would have to create a .env file to fulfill all the needed environment variables, the run `npm run server:dev`
- Though the front end has an option for adding images, the image that you add won't reflect on the centers page and this is because I am yet to add the cloud image hosting service to help handle the image storage. The image that would be displayed would be the default image that I set when rendering the page.
- If you really want an image, you can do so by posting the link to an image when creating a center using just the API endpoint with an API testing tool.
#### What are the relevant pivotal tracker stories?
#152797395 
#### Screenshots (if appropriate)
![screencapture-localhost-8080-editcenter-1513103139280](https://user-images.githubusercontent.com/26048536/33901484-3567a85c-df72-11e7-8728-11c455dd0092.png)
#### Questions:
None
